plugins/module_utils/network/ios/rm_templates/acls.py

```diff
@@ -281,11 +281,11 @@ def __init__(self, lines=None):
         {
             "name": "aces",
             "getval": re.compile(
-                r"""(\s*(?P<sequence>\d+))?
-                        (\s*sequence\s(?P<sequence_ipv6>\d+))?
-                        (\s*(?P<grant>deny|permit))
+                r"""^\s*((?P<sequence>\d+))?
+                        (\ssequence\s(?P<sequence_ipv6>\d+))?
+                        (\s(?P<grant>deny|permit))
                         (\sevaluate\s(?P<evaluate>\S+))?
-                        (\s(?P<protocol_num>\d+))?
+                        (\s(?P<protocol_num>\d+)\s)?
```

Comment:
I don't remember exactly but I think it was put at one time to avoid to match an IP address like a protocol number...
But I didn't have successfully reproduce issues without this space in the regexp so, possibily, it's not required anymore...

Reply:
OK: I have found.

Currently, if you don't put this whitespace, for all standard ACL; you would have an error because, the first digit of the IP address is considere as a "protocol number" and not the first digit of the IP address.

By example, currently, the following ACL is not correctly matched:
```
ip access-list standard acl_test
 deny 10.0.0.0 0.255.255.255
```
(the result was the registering of the ace as an ACE with a protocol  and nothing else.

But I have an error in my ACL (currently, I don't match correctly the real rules with a protocol number as number.
Let me fixe it.

Reply:
OK: the issue is solved in the ACL